lib/auth0/api/v2/client_grants.rb

```diff
@@ -11,13 +11,15 @@ module ClientGrants
         # @param audience [string] The audience of the client grant to retrieve.
         # @param page [int] Page number to get, 0-based.
         # @param per_page [int] Results per page if also passing a page number.
+        # @param allow_any_organization [bool] Optional filter on allow_any_organization.
```

Comment:
Should we expand a bit on what `allow_any_organization` does?

Reply:
I'd agree if you were setting the `allow_any_organization` field, but this is a filter on the value